### PR TITLE
doc: edit PendingReleaseNotes v18 for Reef

### PR DESCRIPTION
**Pull request by zdover23**
Line edit PendingReleaseNotes for Reef.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
jenkins retest this please

**Comment by zdover23** [comment]
Continued in https://github.com/ceph/ceph/pull/52112.

File: PendingReleaseNotes [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+* RGW's default back end for `rgw_enable_ops_log` has changed from ``RADOS`` to
+ ``file``. The default value of ``rgw_ops_log_rados`` is now ``false``, and
+ ``rgw_ops_log_file_path`` defaults to
+ ``/var/log/ceph/ops-log-$cluster-$name.log``.
+* The SPDK back end for BlueStore is now able to connect to an NVMeoF target.
+  This is not an officially supported feature.
+* RGW's pubsub interface now returns boolean fields using ``bool``. Before this
+  change, ``/topics/<topic-name>`` returns ``stored_secret`` and ``persistent``
+  using a string of ``"true"`` or ``"false"``  that contains enclosing quotation
+  marks.  After this change, these fields are returned without enclosing
```
Comment (colemitchell) [inline PendingReleaseNotes:16]: @zdover23 - Somehow a double space was introduced.

File: PendingReleaseNotes [hunk 2]
```diff
@@ -47,120 +50,128 @@
...
+  part of the conversion, many were also renamed in order to better
+  disambiguate journal-based and snapshot-based mirroring.
+* RBD: The list-watchers C++ API (``Image::list_watchers``) now clears the 
+  passed ``std::list`` before appending to it. This aligns with the semantics
+  of the C API (``rbd_watchers_list``).
+* Telemetry: Users who have opted in to telemetry can also opt in to
+  participate in a leaderboard in the telemetry public dashboards
+  (https://telemetry-public.ceph.com/). In addition, users are now able to
+  provide a description of their cluster that will appear publicly in the
+  leaderboard.  For more details, see:
```
Comment (colemitchell) [inline PendingReleaseNotes:141]: @zdover23 - A double space seems to have been introduced during the line wrap.

File: PendingReleaseNotes [hunk 3]
```diff
@@ -47,120 +50,128 @@
...
+  https://docs.ceph.com/en/latest/mgr/telemetry/#leaderboard To see a sample
```
Comment (colemitchell) [inline PendingReleaseNotes:142]: @zdover23 - There needs to be a period after the address.

File: PendingReleaseNotes [hunk 4]
```diff
@@ -47,120 +50,128 @@
...
+  report, run ``ceph telemetry preview``.  To opt in to telemetry, run ``ceph
```
Comment (colemitchell) [inline PendingReleaseNotes:143]: @zdover23 - A double space seems to have been introduced during the line wrap.

File: PendingReleaseNotes [hunk 5]
```diff
@@ -1,26 +1,29 @@
...
-  are:
+* RGW's default back end for `rgw_enable_ops_log` has changed from ``RADOS`` to
+ ``file``. The default value of ``rgw_ops_log_rados`` is now ``false``, and
+ ``rgw_ops_log_file_path`` defaults to
+ ``/var/log/ceph/ops-log-$cluster-$name.log``.
+* The SPDK back end for BlueStore is now able to connect to an NVMeoF target.
+  This is not an officially supported feature.
+* RGW's pubsub interface now returns boolean fields using ``bool``. Before this
+  change, ``/topics/<topic-name>`` returns ``stored_secret`` and ``persistent``
+  using a string of ``"true"`` or ``"false"``  that contains enclosing quotation
```
Comment (colemitchell) [inline PendingReleaseNotes:15]: @zdover23 - Somehow a double space was introduced.

File: PendingReleaseNotes (hunk 1 above)
Comment (zdover23) [reply PendingReleaseNotes:16]: Accepted.

File: PendingReleaseNotes (hunk 2 above)
Comment (zdover23) [reply PendingReleaseNotes:141]: Accepted.

File: PendingReleaseNotes (hunk 3 above)
Comment (zdover23) [reply PendingReleaseNotes:142]: Accepted.

File: PendingReleaseNotes (hunk 4 above)
Comment (zdover23) [reply PendingReleaseNotes:143]: Accepted.

File: PendingReleaseNotes [hunk 6]
```diff
@@ -47,120 +50,128 @@
...
+  of the C API (``rbd_watchers_list``).
+* Telemetry: Users who have opted in to telemetry can also opt in to
+  participate in a leaderboard in the telemetry public dashboards
+  (https://telemetry-public.ceph.com/). In addition, users are now able to
+  provide a description of their cluster that will appear publicly in the
+  leaderboard.  For more details, see:
+  https://docs.ceph.com/en/latest/mgr/telemetry/#leaderboard To see a sample
+  report, run ``ceph telemetry preview``.  To opt in to telemetry, run ``ceph
+  telemetry on``.  To opt in to the leaderboard, run ``ceph config set mgr
+  mgr/telemetry/leaderboard true``.  To add a leaderboard description, run
```
[the remark this answers is not in the document]
Comment (zdover23) [reply PendingReleaseNotes:145]: Accepted.

**Comment by zdover23** [comment]
jenkins retest this please

**Comment by zdover23** [comment]
Superseded by https://github.com/ceph/ceph/pull/52490.

**Pull request closed without merging**